### ISSUE TITLE
Add Input Validations for AddCar View

### ISSUE THREAD

**Issue by luisjrez** [luisjrez/RN-Template-eCommerce-#5]
![Input_Validations](https://user-images.githubusercontent.com/78288250/138803803-d61e92dc-7725-4114-b0e4-6aedf7a33ef5.jpeg)

- Please add validations to the inputs so the user should only enter valid data

**Comment by beheradibas** [comment]
I would like to work on this, but how can I check if the security code is valid or not. I guess we can only check if the card number is valid or not, the security code is confidential.

**Comment by luisjrez** [comment]
@beheradibas can you make a hook to make these validations and you can use `https://github.com/braintree/card-validator` to make the card number validation and cvv validation, you can add plain TextInputs without styles and make the validations works!

**Comment by beheradibas** [comment]
How do I run it locally ? 
Unable to run using `npm install && npm run android`

**Comment by luisjrez** [comment]
@beheradibas yeah that's the way to run locally for android development, you can running locally with that command? what error do you have?

**Comment by beheradibas** [comment]
I was trying to run `npm i` in the parent folder, tried in ECommerce directory and it installed dependencies.
But for running the app, I am getting errors suggesting to install android development environment setup.
Guide me on that.
